src/x_midi.c

```diff
@@ -586,11 +586,21 @@ static void midiout_float(t_midiout *x, t_floatarg f)
     outmidi_byte(x->x_portno - 1, f);
 }
 
+static void midiout_list(t_midiout *x, t_symbol *s, int ac, t_atom *av)
+{
+    for (int i = 0; i < ac; ++i)
```

Comment:
this is only valid in C99; since adding support for ANSI C is pretty straightforward, i would suggest to use:
~~~C
int i;
for (i = 0; i < ac; ++i)
{
   // ..
}
~~~

Reply:
Got it. Is C99 still a hard limit? I suppose so if we want to continue supporting Windows XP builds? ;P